layouts/partials/widgets/social.html

```diff
@@ -41,6 +41,14 @@ <h4 class="widget-social__title widget__title">{{ T "social_title" }}</h4>
 			</a>
 		</div>
 		{{- end }}
+		{{- with .Site.Params.widgets.social.gitlab }}
+		<div class="widget-social__item widget__item">
+			<a class="widget-social__link widget__link btn" title="GitLab" rel="noopener noreferrer" href="https://gitlab.com/{{ . }}" target="_blank">
+				<svg class="widget-social__link-icon icon-github" viewBox="0 0 265 265" width="24" height="24" fill="#fff"><path d="M128.075 236.075l47.104-144.97H80.97l47.104 144.97z M14.956 91.104L.642 135.16a9.752 9.752 0 0 0 3.542 10.903l123.891 90.012-113.12-144.97z M14.956 91.105H80.97L52.601 3.79c-1.46-4.493-7.816-4.492-9.275 0l-28.37 87.315z M241.194 91.104l14.314 44.056a9.752 9.752 0 0 1-3.543 10.903l-123.89 90.012 113.119-144.97z M241.194 91.105h-66.015l28.37-87.315c1.46-4.493 7.816-4.492 9.275 0l28.37 87.315z"/></svg>
```

Comment:
`icon-github` -> `icon-gitlab`

Reply:
Made the changes you requested and then I squashed that commit.